Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -577,80 +595,84 @@ public class RestRequest {
                 }
                 let fileManager = FileManager.default
                 try fileManager.moveItem(at: source, to: destination)
-
+                
                 completionHandler(response as? HTTPURLResponse, error)
-
+                
             } catch {
                 completionHandler(nil, RestError.fileManagerError)
             }
         }
         task.resume()
     }
-
+    
     /// Method used by `CircuitBreaker` as the contextCommand
     ///
     /// - Parameter invocation: `Invocation` contains a command argument, Void return type, and a String fallback arguement
-    private func handleInvocation(invocation: Invocation<(Data?, HTTPURLResponse?, Error?) -> Void, Void, String>) {
+    private func handleInvocation(invocation: Invocation<(Data?, HTTPURLResponse?, Error?) -> Void, String>) {
         let task = session.dataTask(with: request) { (data, response, error) in
             if error != nil {
-                invocation.notifyFailure()
+                invocation.notifyFailure(error: BreakerError.fastFail)
```

Comment:
This shouldn't be a fastFail error. This is reserved for errors caused by the circuit being open; instead, we should  create a custom BreakerError and pass that in
```
invocation.notifyFailure(error: BreakerError(reason: error.description))
```